app/views/users/sessions/new.html.erb

```diff
@@ -1,28 +1,32 @@
 <% if params[:restricted] %>
-  <h3 class="heading"><%= ts('Sorry!') %></h3>
+  <h3 class="heading"><%= ts("Sorry!") %></h3>
   <p>
-    <%= ts('This work is only available to registered users of the Archive.') %>
-    <%= ts('If you already have an Archive of Our Own account, log in now.') %>
-    <% if @admin_settings.account_creation_enabled? %>
-      <%= link_to ts("Or join us for free - it's easy."), signup_path %>
+    <%= ts("This work is only available to registered users of the Archive.") %>
+    <%= ts("If you already have an Archive of Our Own account, log in now.") %>
+      <% if @admin_settings.invite_from_queue_enabled? && @admin_settings.creation_requires_invite? %>
+        <%= ts("If you don't have an account, you can") %> <%= link_to ts("request an invitation"), invite_requests_path %> <%= ts("to join.") %>
```

Comment:
Unfortunately, Translation can't work with sentences that are broken into separate pieces like this, since sentences in different languages will be structured differently and might not always break into three parts with a link in the middle. 

Can you change over to the style we use here?
https://github.com/otwcode/otwarchive/blob/ec0517e0de66fe39a83046b3c6bb7066b450a6ea/app/views/comments/_commentable.html.erb#L85

Here's the corresponding part of the locale file for the above as well: 
https://github.com/otwcode/otwarchive/blob/ec0517e0de66fe39a83046b3c6bb7066b450a6ea/config/locales/views/en.yml#L100-L104

Reply:
okay, just to make sure, i'll be doing this for lines 7 and 18 which are broken into 3 pieces

`<%= ts("If you don't have an account, you can") %> <%= link_to ts("request an invitation"), invite_requests_path %> <%= ts("to join.") %>`

should i also do it for the lines that are broken into 2 pieces like...

`<%= ts("Don't have an account?") %> <%= link_to ts("Create an account now."), signup_path %>`

or lines like this one?

`<%= link_to ts("Or join us for free - it's easy."), signup_path %>`

Reply:
Oh, yeah, the ones in two pieces definitely need it as well! I didn't notice those, so good catch.  (Actually, everything needs to be converted eventually, so if you're in the mood, have at it!)

Reply:
errr i am very iffy about the way i structured this in `en.yml` /o\ it seems to work but i wasn't exactly sure how to best name everything...

Reply:
noooooooo not the gitignore changes /o\